ui/serializers.py

```diff
@@ -67,8 +71,5 @@ class DropboxFileSerializer(serializers.Serializer):
     isDir = serializers.BooleanField()
 
     def create(self, validated_data):
-        video, _ = Video.objects.get_or_create(
-            s3_object_key=validated_data["name"],
-            defaults={'source_url': validated_data["link"]},
-        )
+        video, _ = Video.objects.get_or_create(defaults={'source_url': validated_data["link"]},)
```

Comment:
I think this  'create' method was intended to allow for creating a video via the REST API, but it's not actually used in the app anywhere.